rails_programming/rails_basics/deployment.md

```diff
@@ -143,3 +153,14 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Heroku Custom Domains](https://devcenter.heroku.com/articles/custom-domains) help file
 * [Converting from PostgreSQL to SQLite3](http://manuel.manuelles.nl/blog/2012/01/18/convert-postgresql-to-sqlite/)
 * [Webinar on Using Heroku in Production](https://blog.heroku.com/archives/2013/7/11/running-production-apps-on-heroku)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="#dyno-knowledge-check">What are dynos? how do they work?</a>
+* <a class="knowledge-check-link" href="#management-knowledge-check">What are the two ways you can manage your application with heroku?</a>
+* <a class="knowledge-check-link" href="#new-app-knowledge-check">How do you create a new application with heroku?</a>
+* <a class="knowledge-check-link" href="#db-knowledge-check">Which of the database gems are supported by heroku?</a>
+* <a class="knowledge-check-link" href="#asset-error-knowledge-check">What are asset errors? Why do they pop up?</a>
+* <a class="knowledge-check-link" href="#logs-knowledge-check">How do you check the server output?</a>
+* <a class="knowledge-check-link" href="#localhost-knowledge-check">How can you run multiple rails apps at once?</a>
```

Comment:
Could we remove this one? It's nice to know but I don't think it rises to the point of being worth having a knowledge check.